pkgs/top-level/all-packages.nix

```diff
@@ -12743,6 +12743,8 @@ with pkgs;
 
   sqls = callPackage ../applications/misc/sqls { };
 
+  stardrop = callPackage ../by-name/st/stardrop/package.nix { };
```

Comment:
```suggestion
```

Reply:
@jh-devv Please don't mark feedback as resolved if they're unresolved.

Reply:
> @jh-devv Please don't mark feedback as resolved if they're unresolved.

Oh, sorry, GitHub web had some issues and I couldn't commit the changes. I had to make it an "new commit"

-- Fixed with #b301e6b